mint-auditor/src/gnosis/mod.rs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+//! Gnosis safe auditing support.
+
+mod config;
+mod error;
+mod eth_data_types;
+
+pub use config::{AuditedSafeConfig, GnosisSafeConfig};
+pub use error::Error;
+pub use eth_data_types::EthAddr;
```

Comment:
Optional/Equivalent: Use grouped `pub use`
```suggestion
pub use self::{
    config::{AuditedSafeConfig, GnosisSafeConfig},
    error::Error,
    eth_data_types::EthAddr,
};
```